### ISSUE TITLE
Still experience errors such as https://github.com/mitre/caldera/issues/1789

### ISSUE THREAD

**Issue by robertnixon2003** [mitre/caldera#1932]
This is in relation to https://github.com/mitre/caldera/issues/1789 that was auto-closed with no comments.

```
2020-10-23 10:32:09 - ERROR (base_events.py:1707 default_exception_handler) Task exception was never retrieved
future: <Task finished name='Task-34' coro=<build_docs() done, defined at /home/deploy/caldera/plugins/fieldmanual/hook.py:24> exception=FileNotFoundError(2, 'No such file or directory')>
Traceback (most recent call last):
  File "/home/deploy/caldera/plugins/fieldmanual/hook.py", line 25, in build_docs
    process = await asyncio.create_subprocess_exec(
  File "/usr/lib/python3.8/asyncio/subprocess.py", line 236, in create_subprocess_exec
    transport, protocol = await loop.subprocess_exec(
  File "/usr/lib/python3.8/asyncio/base_events.py", line 1630, in subprocess_exec
    transport = await self._make_subprocess_transport(
  File "/usr/lib/python3.8/asyncio/unix_events.py", line 197, in _make_subprocess_transport
    transp = _UnixSubprocessTransport(self, protocol, args, shell,
  File "/usr/lib/python3.8/asyncio/base_subprocess.py", line 36, in __init__
    self._start(args=args, shell=shell, stdin=stdin, stdout=stdout,
  File "/usr/lib/python3.8/asyncio/unix_events.py", line 789, in _start
    self._proc = subprocess.Popen(
  File "/usr/lib/python3.8/subprocess.py", line 854, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.8/subprocess.py", line 1702, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'sphinx-build'
```

**Comment by robertnixon2003** [comment]
Only happens when not using the --insecure flag

**Comment by scottctaylor12** [comment]
Hey @robertnixon2003, I found that the problem is CALDERA is trying to execute the command `sphinx-build` which was installed when installing the python package dependencies. However, the actual binary of the command is stored at `/home/<user>/.local/bin` which is not in the environment PATH.

Therefore add this directory to your path and let me know if it fixes the sphinx build issue! 
Use the following commands:
`export PATH=/home/deploy/.local/bin:$PATH`
`python3 /home/deploy/caldera/server.py`
NOTE: the /home/deploy directory is specific to your issue based on your error log

**Comment by unkempthenry** [comment]
This should be fixed when 2.8.2 is released ( https://github.com/mitre/fieldmanual/pull/20 ).  @scottctaylor12' suggestion will work -- you can also install caldera and it's libraries in a python virtualenv.  Also, if you install sphinx as root this will usually result in the `sphinx-build` binary stub getting added to PATH. For more detail about this see my responses on this related issue https://github.com/mitre/caldera/issues/1817

**Comment by unkempthenry** [comment]
Also, the fieldmanual docs are the same as the docs that are available here https://caldera.readthedocs.io/en/latest/ . If the publicly available docs work for you, you may be able to to disregard the error message until 2.8.2 comes out.
